recipes/annexremote/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "annexremote" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2be77ae9b9edd0d2818ab6dff7070a05aed2fcc1e5065638aa03eba991b67f17
+
+build:
+  skip: True   # [not py3k]
```

Comment:
You can use a version constraint on python below instead of the selector (e.g., `python <3` or whatever). See here: https://conda-forge.org/docs/meta.html#building-noarch-packages

Then you can keep the noarch selector.